tests/langchain/test_langchain_model_export.py

```diff
@@ -84,14 +95,18 @@ def create_openai_llmagent():
     return initialize_agent(tools, llm, agent=AgentType.ZERO_SHOT_REACT_DESCRIPTION, verbose=True)
 
 
-def create_model(llm_type, model_path=None):
-    if llm_type == "openai":
+def create_model(model_type, model_path=None):
```

Comment:
Do we need this function? Can we just directly use each function?

Reply:
I implemented this function for better readability. It's a centralized place for us to easily know which models are being tested, and hide the implementation details for creating each model.
Do you think it is useful in this sense?

Reply:
Makes sense, not a big deal. I'd probably remove it for quicker code navigation:

```python
model = create_model("qaevalchain")
# need to jump to create_model first,
# then find a matching if branch
# then jump to the matched function
```

```python
model = create_openai_llmagent()
# can directly jump to `create_openai_llmagent`
```

Reply:
Yeah, actually now I also feel it's more convenient to remove this function :)